src/data/species-form-change-triggers/species-form-change-status-effect-trigger.ts

```diff
@@ -16,6 +16,6 @@ export class SpeciesFormChangeStatusEffectTrigger extends SpeciesFormChangeTrigg
   }
 
   override canChange(pokemon: Pokemon): boolean {
-    return this.statusEffects.indexOf(pokemon.status?.effect || StatusEffect.NONE) > -1 !== this.invert;
+    return this.statusEffects.indexOf(pokemon.getStatusEffect(true) || StatusEffect.NONE) > -1 !== this.invert;
```

Comment:
```suggestion
    const hasStatus = pokemon.hasStatusEffect(this.statusEffects, false, true)
    return this.invert ? hasStatus : !hasStatus;
```
I think this is equivalent? This doesn't seems to be used by any Pokemon so...